### PR TITLE
Sort arcs by story order instead real date

### PR DESCRIPTION
**Pull request by derpynirvash**
Some basic code to allow arcs to be sorted by a value, time_line in the header of each file.

### CODE REVIEW

File: .themes/default/source/arcs.html [hunk 1]
```diff
@@ -15,8 +15,6 @@ <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slug
       {% include rp.html rp=arc_rp %}
     {% endfor %}
     </ul>
-  {% else %}
-    {% include rp.html rp=rp %}
```
Comment (irisfofs) [inline .themes/default/source/arcs.html:19]: Why remove this? Now RPs not part of an arc won't be displayed on the arc page.

Comment (derpynirvash) [inline .themes/default/source/arcs.html:19]: It is an index of all arcs, why would you put non arc'ed rps on the page? Leads to a lot of clutter.

Comment (irisfofs) [inline .themes/default/source/arcs.html:19]: Well, it's more of a display that displays arcs when they exist, and also other ones. I dunno, haven't really thought about it.

I'd rather not change it now as part of this, that should be its own thing.

Comment (derpynirvash) [inline .themes/default/source/arcs.html:19]: Eh, in my opinion, it at least needs to be sorted better.

But sounds fine, leaving design decisions for later.

Comment (irisfofs) [inline .themes/default/source/arcs.html:19]: Can you make an issue for the Arc View sorting being confusing / needing to
be improved?

**Comment by derpynirvash** [comment]
Style stuff was like, removed, man.

**Comment by irisfofs** [comment]
Yeah, sorry, I was at a con. :P I want to do a bit of tweaking to stuff once I merge it, so I have held off until I'm ready to do that.

**Comment by derpynirvash** [comment]
As requested, change the way that the arcs get sorted so that the timeline value will just override the start date of the file.

**Comment by irisfofs** [comment]
(thought it was actually merged, just... more complicatedly.)

**Pull request closed without merging**